### PR TITLE
[vcpkg baseline][vcpkg scripts] Drop `${pc_sysrootdir}`

### PR DESCRIPTION
**Pull request by dg0yt**
Fixes xcb-util-errors baseline regression.

### CODE REVIEW

File: ports/xcb-proto/portfile.cmake [hunk 1]
```diff
@@ -38,6 +38,7 @@ vcpkg_configure_make(
         ac_cv_path_PYTHON='${PYTHON3}'
         am_cv_python_pyexecdir=\\\${prefix}/tools/python3/site-packages
```
Comment (Neumann-A) [inline ports/xcb-proto/portfile.cmake:39]: If you already touch the file please change this to `am_cv_python_pyexecdir=\\\${prefix}/${PYTHON3_SITE}`

File: ports/xcb-proto/portfile.cmake [hunk 2]
```diff
@@ -38,6 +38,7 @@ vcpkg_configure_make(
         ac_cv_path_PYTHON='${PYTHON3}'
         am_cv_python_pyexecdir=\\\${prefix}/tools/python3/site-packages
         am_cv_python_pythondir=\\\${prefix}/tools/python3/site-packages
+        "XCBPROTO_XCBINCLUDEDIR=${CURRENT_INSTALLED_DIR}/share/xcb-proto/xcb"
```
Comment (Neumann-A) [inline ports/xcb-proto/portfile.cmake:41]: So you want to override pc_sysrootdir if I remember correctly. Maybe leave a comment that this is the aim? 

I used the following:
```
vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/lib/pkgconfig/xcb-proto.pc" "\${pc_sysrootdir}" "")
if(NOT VCPKG_BUILD_TYPE)
  vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/debug/lib/pkgconfig/xcb-proto.pc" "\${pc_sysrootdir}" "")
endif()
```

**Comment by dg0yt** [comment]
x64-osx hangs at "Performing post-build validation" for python2 :-(

**Comment by dg0yt** [comment]
@BillyONeal et al. Can you cancel/re-run x64-osx without re-triggering the other triplets? Don't want to rebuild llvm more often than needed.

**Comment by BillyONeal** [comment]
> @BillyONeal et al. Can you cancel/re-run x64-osx without re-triggering the other triplets? Don't want to rebuild llvm more often than needed.

Unfortunately as far as I know one can't cancel individual triplets like that :(

**Review by BillyONeal — APPROVED** [review]

**Review by FrankXie05 — APPROVED** [review]

**Pull request merged**